src/airtable/constants/earmark-type-map.constant.ts

```diff
@@ -0,0 +1,9 @@
+import { EarmarkTypeEnum } from '../../database/enums/earmark-type.enum';
+
+export const EarmarkTypeMap = {
+    'Outreach': EarmarkTypeEnum.Outreach,
+    'New Outreach': EarmarkTypeEnum.NewOutreach,
+    'New Entrants': EarmarkTypeEnum.NewEntrants,
+    'Core Tech': EarmarkTypeEnum.CoreTech,
+    'New General': EarmarkTypeEnum.NewGeneral,
```

Comment:
question(non-blocking): should we hard-code the earmark types here? 

I think that the DAO will change the types on a regular basis. The best approach would probably be to store the types for every round in the database. This would make it very easy to handle varying earmarks if OceanDAO changes anything.

Reply:
not sure if i understand this question, this is just a translation map to converse the airtable values the ones we use. The earmarks will be saved in our database within the round collection.